app/locales/tr.json

```diff
@@ -175,5 +175,9 @@
   "Close": "Kapat",
   "Contribute now": "Şimdi destek ol",
   "Help us uncover and predict the corona spread in your area": "Bölgendeki corona yayılımını analiz edebilmemiz için bize destek ol",
-  "Numbers": "Sayılar"
+  "Numbers": "Sayılar",
+  "In isolation due to orders from the government": "In isolation due to orders from the government",
```

Comment:
"In isolation due to orders from the government": "Hükümet tarafından alınan kararlar dolayısıyla karantinada",
  "In isolation due to COVID-19 infection": "COVID-19 bulaşması yüzünden karantinada",
  "Some questions and alternatives were not a part from the beginning:": "Bazı sorular ve alternatifler en başından beri yoktu:",

@btutal  can proof read and edit the file as a contributor.

Reply:
Thanks @sariug , I've just commited Turkish trnslations